api-guidelines/global/core-principles/documentation.md

```diff
@@ -1,57 +1,39 @@
 # Documentation
 
-Our [quality standards](./quality-standards.md) define all the values that are important to us.
-What's more, we aim to take care of clearly structured documentation, provide our audience with code examples, and make sure they have a great time using our API.
-Here's what that means:
+We strive to make API implementation a breeze with meaningful, easy-to-understand documentation.
+This includes, but is not limited to:
 
-:::: accordions
-::: accordion Clear structure
-**Usability**
+- [Accessibility and usability](#accessibility-and-usability)
+- [Code examples](#code-examples)
+- [Context and walkthroughs](#context-and-walkthroughs)
+- [Up-to-dateness](#up-to-dateness)
 
-- A user-friendly navigation makes it easy to discover the API and find all relevant information.
-- The sidebar is cleary structured and easy to navigate.
-- API endpoints and examples are shown in context.
-
-**Getting started**
-
-- Basic information is available at one central place, such as info about HTTP methods, API operations, request/response headers, versioning, pagination, or errors.
-- A short and simple guide provides an overview of the most likely tasks to perform with the API.
-- Users can get in touch with us via a contact form.
-
-**Tutorials**  
-With step-by-step walkthroughs we'll cover specific functionality that can be implemented with the API.
-
-**Familiarize users with the API**  
-We'll cater for experienced users that already have an idea of the endpoints to be used, and at the same time provide the respective structure and information for inexperienced users who need a thorough introduction and/or example use cases.
+## Accessibility and usability
 
-**Discoverable and easy to digest**
-
-- All information is easily discoverable, easy to digest, and prepared in a way that the user can effectively work with it.
+- We provide access to all API endpoints in a central location, some sort of documentation portal at the best, to ensure that users can easily use the API.
+- The documentation is browsable.
+- A well-structured sidebar ensures that the documentation can be navigated so that all the endpoints can be accessed.
```

Comment:
Somewhat duplicated by "All information is easily discoverable, easy to digest, and prepared in a way that users can effectively work with it." Maybe remove it or at least shorten it to focus on the sidebar?

```suggestion
- A well-structured sidebar ensures that the documentation can be easily navigated.
```